src/pretalx/cfp/templates/cfp/event/cfp.html

```diff
@@ -43,6 +43,13 @@ <h1>{{ cfp.headline|default:"" }}</h1>
                     {% translate "Edit or view your proposals" %}
                 </a>
             {% endif %}
+
+            {% if has_submissions or request.user.is_anonymous %}
+            <a class="btn btn-info btn-lg btn-block" href="{{ request.event.urls.user }}">
+                {% translate "View or edit speaker profile" %}
+            </a>
+        {% endif %}
```

Comment:
```suggestion
            {% if has_submissions or request.user.is_anonymous %}
                <a class="btn btn-info btn-lg btn-block" href="{{ request.event.urls.user }}">
                    {% translate "View or edit speaker profile" %}
                </a>
            {% endif %}
```

Please indent appropriately.